tests/document-tests.sh

```diff
@@ -54,7 +56,12 @@ if test "`ls -d pwg-raster-samples-*dpi-20150616 2>/dev/null`" = ""; then
 	exit 1
 fi
 
-$IPPFIND "$1._ipp._tcp.local." -x $IPPTOOL -P "$1 Document Results.plist" -I '{}' document-tests.test \;
+
+PLIST="$1 Document Results $(date +'%Y%m%d%H%M').plist"
+$IPPFIND --name "^$1$" "_ipp._tcp.local." -x $IPPTOOL -P "$PLIST" -I '{}' document-tests.test \;
```

Comment:
Need \$ for regex. Maybe define variable "NAME" above as regular expression and use "$NAME" here?